client/src/GamemasterDesktop/DesktopView.tsx

```diff
@@ -72,9 +92,11 @@ function Grimoire({ isPlayerView = true }: GrimoireProps) {
                 <TeamDistributionBar />
                 <div className="text-center">
                   <div>Alive players: {alivePlayers.length}</div>
-                  <div>{`Votes to execute: ${Math.ceil(
-                    alivePlayers.length / 2,
-                  )}`}</div>
+                  <div className="capitalize">{`Currently executeing: ${
+                    playerOnBlock.player ?? "Nobody"
+                  }`}</div>
+                  <div>{`Votes needed : ${playerOnBlock.votes}`}</div>
```

Comment:
I find "Votes needed" just slightly confusing, since it can be - but isn't necessarily - votes to tie.